### DISCUSSION TITLE
blog/inputs-service/

### DISCUSSION THREAD

**Discussion by giscus** [nartc/tnc-scully#44] [Announcements]
# blog/inputs-service/

Personal blog by Chau Tran

https://nartc.me/blog/inputs-service/

**Comment by minhquoc13** [comment]
> nice

**Comment by id1945** [comment]
![nice](https://i.ytimg.com/vi/Iwus8XOU_hA/sddefault.jpg)

**Comment by amosISA** [comment]
What do you think about JSX in Angular?
